docs/src/pages/components/paper/paper.md

```diff
@@ -27,3 +27,7 @@ If you need an outlined surface, use the `variant` prop.
 The elevation can be used to establish a hierachy between other content. In practical terms, the elevation controls the size of the shadow applied to the surface. In dark mode, raising the elevation also makes the surface lighter.
 
 {{"demo": "pages/components/paper/Elevation.js", "bg": "inline"}}
+
+The change of shade in dark mode is done by applying a semi-transparent gradient to the `background-image` property.
+It may be confusing when overriding the Paper's styles, as setting just the `background-color` will not affect the elevation-related shading.
```

Comment:
```suggestion
This can lead to confusion when overriding the styles of `Paper`, as setting just the `background-color` property will not affect the elevation-related shading.
```